WalletWasabi.Gui/Tabs/WalletManager/LoadWalletView.xaml

```diff
@@ -68,7 +70,17 @@
           </StackPanel>
         </Grid>
         <Grid Classes="content">
-          <ListBox Items="{Binding Wallets}" SelectedItem="{Binding SelectedWallet, Mode=TwoWay}" />
+          <ListBox Items="{Binding Wallets}" SelectedItem="{Binding SelectedWallet, Mode=TwoWay}">
+            <ListBox.ItemTemplate>
+              <DataTemplate>
+                <TextBlock Text="{Binding WalletName}">
+                  <i:Interaction.Behaviors>
+                    <behaviors:CommandOnDoubleClickBehavior Command="{Binding $parent[ListBox].DataContext.LoadCommand}" />
```

Comment:
I would prefer this more wpf-like:
`C# Command="{Binding RelativeSource={RelativeSource AncestorType=ListBox}, Path=DataContext.LoadCommand}"`

Reply:
I agree, it looks much better.